common/source/docs/common-nooploop.rst

```diff
@@ -0,0 +1,87 @@
+
+Placing the anchors
+===================
+
+The anchors should be placed in a rectangular shape. The anchor 0 will act as "origin" and Y axis of Nooploop system will act as "North". The configuration is slightly simpler if the line from the anchor 0 to the anchor 1 is due North but it is not required as the :ref:`BCN_ORIENT_YAW <BCN_ORIENT_YAW>` parameter can be used to account this difference.
+
+.. image:: ../../../images/nooploop-anchor-layout.png
+    :target: ../_images/nooploop-anchor-layout.png
+
+Preparing the tag
+=================
+
+Use Nooploop NAssistant tool to configure one node to act as a tag.
+
+- set Protocol to Node_Frame2
+- set Baudrate to 921600
+- set Update Rate to 25
+- set Filter Factor to 0 
+
+.. image:: ../../../images/nooploop-config-tag.png
+    :target: ../_images/nooploop-config-tag.png
+
+Connecting the tag to the Flight Controller
+===========================================
+
+It is possible to directly connect UART port of the tag to TELEM2 port of the FC. However, UART Tx of certain FCs is not compatible with Nooploop UART Rx. It causes the FC unable to communicate with the tag. Pixhawk 4 mini and Kakute F7 is confirmed that they have this issue. In this case, a Arduino Nano Every can be used as a work-around. Use Ardnio IDE to upload `Nooploop work-around sketch <https://github.com/chobitsfan/nooploop_workaround.git>`__ to a Arduino Nano Every and connect its UART Tx to UART Rx of the tag.
+
+.. image:: ../../../images/nooploop-connect-tag.png
+    :target: ../_images/nooploop-connect-tag.png
+
+Configuration through the Ground Station
+========================================
+
+Ensure that Copter-4.1 (or higher) is loaded onto the autopilot and connect with a Ground Station (i.e. Mission Planner).
+
+- set :ref:`BCN_TYPE <BCN_TYPE>` to 3 (means using Nooploop system)
+- set :ref:`BCN_LATITUDE <BCN_LATITUDE>`, :ref:`BCN_LONGITUDE <BCN_LONGITUDE>` and :ref:`BCN_ALT <BCN_ALT>` to match your actual location.  Getting these values exactly correct is not particularly important although getting it close is required in order for the compass's declination to be correctly looked up from the small database held within ArduPilot.
+- set :ref:`BCN_ORIENT_YAW <BCN_ORIENT_YAW>` to the heading from the origin anchor to the 2nd anchor.  One way to capture this value is to stand at the origin holding the vehicle so that its nose points towards the second beacon.  Read the vehicle's heading from the HUD and enter this value into :ref:`BCN_ORIENT_YAW <BCN_ORIENT_YAW>`
+- set :ref:`EK3_BCN_DELAY <EK3_BCN_DELAY>` to 20
+- set :ref:`GPS_TYPE <GPS_TYPE>` to 0 to disable the GPS
+- set :ref:`ARMING_CHECK <ARMING_CHECK>` to -9 to disable the GPS arming check
+- set :ref:`SERIAL1_BAUD <SERIAL1_BAUD>` to 921 to set telemetry1's baud rate to 921600 (if using Telem2 set :ref:`SERIAL2_BAUD <SERIAL2_BAUD>` instead)
```

Comment:
SERIAL1 and 2 do not necessarily correspond to TELEM1 and 2 or even UART1 and 2....we usually say before that " for the following we assume that the beacon is connected to the UART port corresponding to SERIAL1" ...then drop the parenthetical comment about using TELEM2